src/components/card/card.stories.mdx

```diff
@@ -0,0 +1,375 @@
+</Preview>
+
+## more `<CardFooter />` examples
+
+<Preview>
+  <Story name="more exampels of CardFooter">
+    <div>
+      <Card>
+          <CardFooter p={1}>
+            <Box alignItems="center" width="100%" display="flex" justifyContent="space-around">
+              <Box flexGrow={1}>
+                <IconButton> 
+                  <Icon bgSize="medium" type="phone"/>
+                </IconButton>
+                <IconButton> 
+                  <Icon bgSize="medium" type="phone"/>
+                </IconButton>
+                <IconButton> 
+                  <Icon bgSize="medium" type="phone"/>
+                </IconButton>
+                <IconButton> 
+                  <Icon bgSize="medium" type="phone"/>
+                </IconButton>
+              </Box>
+              <Box>
+                <Button buttonType="tertiary"> Button </Button>
+                <Button buttonType="primary"> Button </Button>
+              </Box>
+            </Box>
+          </CardFooter>
+        </Card>
+        <Card cardWidth="400px">
+          <CardFooter px={2} py={1}>
+            <Box width="100%" alignItems="center" display="flex" justifyContent="center">
+              <Button p={0} iconPosition="after" iconType="edit" buttonType="tertiary">
+                Edit Button
+              </Button>
+              <VerticalDivider tint={80} py={0} px={2} h={30} /> 
+              <Button p={0} iconPosition="after" iconType="edit" buttonType="tertiary">
+                Edit Button
+              </Button>
+              <VerticalDivider tint={80} py={0} px={2} h={30} />
+              <Button p={0} iconPosition="after" iconType="edit" buttonType="tertiary">
+                Edit Button
+              </Button> 
+              </Box>
+          </CardFooter>
+        </Card>
+        <Card cardWidth="400px">
+          <CardFooter variant="transparent" px={2} py={1}>
+            <Box width="100%" alignItems="center" display="flex" justifyContent="center">
+              <Button p={0} iconPosition="after" iconType="edit" buttonType="tertiary">
+                Edit Button
+              </Button>
+              <VerticalDivider tint={80} py={0} px={2} h={30} /> 
+              <Button p={0} iconPosition="after" iconType="edit" buttonType="tertiary">
+                Edit Button
+              </Button>
+              <VerticalDivider tint={80} py={0} px={2} h={30} />
+              <Button p={0} iconPosition="after" iconType="edit" buttonType="tertiary">
+                Edit Button
+              </Button> 
+              </Box>
+          </CardFooter>
+        </Card>
+        <Card>
+          <CardFooter p={2}>
+            <Box display="flex" width="100%" justifyContent="center">
+              <Link icon="link" href="https://carbon.sage.com/">
+                View Stripe Dashboard
+              </Link>
+            </Box>
+          </CardFooter>
+        </Card>
+      </div>
+  </Story>
+</Preview>
+
+<Props of={Card} />
```

Comment:
We should display the other Card components props here as well (ie CardFooter, CardRow and so on)